poker-pals/src/components/Poker.vue

```diff
@@ -201,13 +203,13 @@ export default {
             // end of socket.on functions
             // asign the standard card_back image to each of player cards inititally
             // later the server will send the true cards at the show down
-            this.myCards = [{src: this.imageFiles.find(file => file.name === 'card_empty').src}, {src: this.imageFiles.find(file => file.name === 'card_empty').src}];
+            this.myCards = [{src: this.imageFiles.getImage('card_empty').src}, {src: this.imageFiles.getImage('card_empty').src}];
             for(let i = 0; i < this.players.length; i++){
-                this.players[i].accountImage.src = this.imageFiles.find(file => file.name === 'invisible').src;
-                this.players[i].cards = {card1: {src: this.imageFiles.find(file => file.name === 'invisible').src}, card2: {src: this.imageFiles.find(file => file.name === 'invisible').src}};
+                this.players[i].accountImage.src = this.imageFiles.getImage('invisible').src;
+                this.players[i].cards = {card1: {src: this.imageFiles.getImage('invisible').src}, card2: {src: this.imageFiles.getImage('invisible').src}};
```

Comment:
I'm confused by the point of the 'invisible' image.

Reply:
there was a timing issue with 'vue -if' for showing component images. it would delay and cause the cards to flicker on reveal. i added this as a redundancy to fix it